src/lib.rs

```diff
@@ -163,41 +161,44 @@ impl UPTransportZenoh {
         let authority = if uri.authority_name.is_empty() {
             self.get_authority()
         } else {
-            uri.authority_name.clone()
+            uri.authority_name()
         };
-        // ue_id
-        let ue_id = if uri.has_wildcard_entity_type() {
-            // If entity type is wildcard, the whole entity should be wilrdcard
+        // ue_type
+        let ue_type = if uri.has_wildcard_entity_type() {
+            "*".to_string()
+        } else {
+            format!("{:X}", uri.uentity_type_id())
+        }
+        .to_string();
```

Comment:
IMHO this is not necessary, is it?

Reply:
Yes, I forgot to remove while doing some tests...